test/unitTests/core/zoe/contracts/test-simpleOfferSwap.js

```diff
@@ -0,0 +1,139 @@
+import { test } from 'tape-promise/tape';
+import harden from '@agoric/harden';
+
+import { makeZoe } from '../../../../../core/zoe/zoe/zoe';
+import { makeSimpleOfferMaker } from '../../../../../core/zoe/contracts/simpleOffer/simpleOffer';
+import { swapSrcs } from '../../../../../core/zoe/contracts/simpleOffer/srcs/swapSrcs';
+import { setup } from '../setupBasicMints';
+
+test('zoe.makeInstance with simpleOffer with swapSrcs', async t => {
```

Comment:
There should be a test that re-uses a zoe instance.

Reply:
What kind of re-use?

Reply:
The main use of a zoe is to make an instance of a contract with `makeInstance`. Zoe is supposed to be long-lived and ready to support multiple instances of multiple contracts. I'd like to see a test that different contract instances don't interfere with each other.

Reply:
Ah, that makes sense. Sure, I can do that

Reply:
Ok, I added a test to the automaticRefund tests that creates three automatic refunds from the same Zoe and sends one offer to each. I changed automaticRefund to keep a count of the offers it receives such that the test can check that the offer count is 1 for each instance. 

It's not rigorous but should do for now I think. Dean had an idea for how Zoe should work that would move to eliminating the "zoeInstance" and just having the Zoe and the governing contract, so it probably doesn't make sense to put all our effort into proving things now.